### ISSUE TITLE
Firewall alias section broken

### ISSUE THREAD

**Issue by madscientist159** [mwood77/pf2opn#21]
The entire alias section of the firewall configuration seems to be unparsed / not updated.

What it should be:

```
          <alias uuid="72b6208b-e50d-4d68-a594-da0fa6760000">
            <enabled>1</enabled>
            <name>MY_HOST</name>
            <type>host</type>
            <proto/>
            <interface/>
            <counters>0</counters>
            <updatefreq/>
            <content>19192.168.1.1</content>
            <categories/>
            <description>Entry added Mon, 01 Jan 1979 01:23:45 -0000</description>
          </alias>
```

What comes out:

```
    <alias>
      <name>MY_HOST</name>
      <type>host</type>
      <address>192.168.1.1</address>
      <descr></descr>
      <detail>Entry added Mon, 01 Jan 1979 01:23:45 -0000</detail>
    </alias>
```

**Comment by mwood77** [comment]
Good to know - i'll look into this over the weekend.

The CDATA #20 might take more time to track down though.

**Comment by madscientist159** [comment]
Since we run OPNsense at an enterprise level I figured our configuration would probably give this tool a good workout.  I'll be filing bugs here and there for specific issues I ran into; for CDATA we have an internal means to deal with it at the moment but that's definitely not something that's ever getting released ;)

**Comment by mwood77** [comment]
This has been addressed in #23 ; closing. If issues persist, please re-open.
